test/ServiceFabric.Mocks.NetCoreTests/ServiceFabric.Mocks.NetCoreTests.csproj

```diff
@@ -4,7 +4,7 @@
     <Description>ServiceFabric.Mocks contains Mock classes to enable unit testing of Actors and Services</Description>
     <Copyright>2022</Copyright>
     <AssemblyTitle>ServiceFabric.Mocks.NetCoreTests</AssemblyTitle>
-    <VersionPrefix>1.0.1</VersionPrefix>
+    <VersionPrefix>1.0.2</VersionPrefix>
```

Comment:
Please revert this, we're not versioning the test dll.

Reply:
done